less/textinput.less

```diff
@@ -3,7 +3,7 @@
     padding: @item-padding;
     position: relative;
     border: @item-border;
-    margin-bottom: 2px;
+    margin-bottom: @item-margin-bottom;
```

Comment:
It looks like `@item-margin-bottom` was introduced as a new variable from the Framework update version 1 to 2. It is only defined [here](https://github.com/adaptlearning/adapt-contrib-vanilla/blob/master/less/paddings.less#L40) and does not exist in the [adapt-defaluts](https://github.com/adaptlearning/adapt-contrib-vanilla/blob/master/less/src/adapt-defaults.less) file left for compatibility with older themes.

If we introduce this variable in a core plugin, it might break building if the variable does not exist in the users theme.
I would recommend to do this styling in the theme directly and not in the plugins less.